leshan-core/src/main/java/org/eclipse/leshan/senml/json/jackson/SenMLJsonRecordSerDes.java

```diff
@@ -171,7 +172,7 @@ public SenMLRecord deserialize(JsonNode o) throws JsonException {
 
         JsonNode vd = o.get("vd");
         if (vd != null && vd.isTextual()) {
-            record.setOpaqueValue(Base64.decodeBase64(vd.asText()));
+            record.setOpaqueValue(new DefaultBase64Decoder(true, true).decode(vd.asText()));
```

Comment:
Should we catch the `IllegalArgumentException` and raise a `JsonException` instead ?